### DISCUSSION TITLE
Who's hiring (March 2023)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#46889] [Show and tell]
# What is this?
This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

# Companies
Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

## Sample Job Post

Here's a good example job post.

> Vercel | Senior Full-Stack Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers/senior-full-stack-engineer-amer)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

If you hire someone through this job board, please reach out to me (hassan@vercel.com) and let me know so I can send you some swag :)

If you're a developer looking to post about themselves, use #45532.

*If you're looking for last month's discussion, see #45533. Several companies [hired someone](https://twitter.com/nutlope/status/1610667757565083648) as a result of it!*

**Comment by anky2k** [comment]
Fox | Senior Front-End Engineer | Remote (IN, UK) | Full-time | [Link](https://www.foxcareers.com/Search/JobDetail/R50020602/senior-front-end-engineer-platform-blockchain-creative-labs)

BCL a Fox company is hiring across the board on various engineering teams and positions, but the above link is for our senior front end engineering position. At Blockchain Creative Labs, we get it. We’re on the frontlines of a monumental shift ourselves. And we’re offering movers and shakers an opportunity to be among the first to control their stake in the new era of blockchain technology. Our team combines industry experience, marketing and business pedigree with an expert engineering team to help all creators and their audiences connect and grow together in the world of Web3. The possibilities are only limited by creativity. Which means, there are no limits.

I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me [directly](https://www.linkedin.com/in/ankit-agarwal-04a01523/).

**Reply by zhyd1997** [reply]
Hi, @anky2k 

Please check my portfolio: https://github.com/vercel/next.js/discussions/46890#discussioncomment-5235598

**Reply by crackwebai1024** [reply]
Hi, @anky2k
I am very interested in your job offer.
Here is my portfolio https://mingxing.vercel.app

**Reply by bluedream1230** [reply]
Dear @anky2k
As an expert, I can say no problem.
[Rich.docx](https://github.com/vercel/next.js/files/10993324/Rich.docx)

This is my CV.
Regards

**Reply by lizhangshengdev** [reply]
Hi @anky2k 
l want to work with you

**Reply by ekil1100** [reply]
Hi, @anky2k

Truly interested in the job position, here is my [resume](https://resume.ekil.io/).

**Comment by dtoyoda-dev** [comment]
Dear,  @anky2k 
I hope your everything goes well.

I am very interested in your job position as senior full stack developer.
Looking forward to work with you. 😎
Best regards

**Comment by brainartfu** [comment]
Hello @anky2k. How are you?
As I am professional fullstack developer, I am very exciting your job position.
Can I work with you and team members? I have good team work experience so I can cooperate with your team closely.
I hope to work with you.
Best regards

**Comment by RecruiterRiemer** [comment]
Capstone IT | Software Engineer | Remote (US only, CST preferred) | W2, C2H | 60-70/hr, salary up to 120k

I'm working with a Fortune 500 company to hire several full stack developers skilled in Java, JavaScript, Vue (or React/Angular), Spring Boot, and SQL. Our client is looking for people with 5+ years of experience, confident in their ability to jump in and start actively coding on day one. Please let me know if you're interested by emailing kristina.riemer@capstonec.com!

**Reply by Eagle9461** [reply]
Hi! I'm interested in your suggestion.
I have 6+ years of experience in Full Stack Development and some desktop application development. 
We can discuss more in skype and google chat.
My email address is ioannis0823glass@gmail.com.
Regards.

**Reply by lizhangshengdev** [reply]
Hi! I'm interested in your suggestion.
I have 4+ years of experience in Full Stack Development and some desktop application development.
We can discuss more in skype and google chat.
My email address is lzs2609534520@gmail.com
Regards.

**Comment by allang** [comment]
Stealth | Full-stack Developer | Remote | Freelance | $50-$80/hr

**Our Project:** We are looking for a freelance full-stack developer who will build a functional MVP with multiple API integrations, including OpenAI’s GPT. The freelancer will be working with a NextJS app with a keen eye for product experience. 

**Requirements:** You will help us deliver our first early version within 2 weeks. The functional requirements are well-defined, small, and tightly scoped. This will be the first of many sprints. You must have a strong interest in the AI space and have experience with GPT. 

**Team:** You will be working closely with high performing designers from top teams in tech. The quality bar is high. The development pace will be fast. 

Please email allang@gmail.com with a link to your Github and 3 recent projects, your roles and responsibilities, and the development durations for these projects.

**Reply by ibrahimyaacob92** [reply]
Hi @allang, I just sent you an email. Looking forward to your response.

**Reply by rodinaleksandr0219** [reply]
Hello @allang. 

I am interested in this position so would like to discuss more details with you if possible.
I attached the repository for testing.
https://github.com/rodinaleksandr0219/ruleev-test
https://github.com/rodinaleksandr0219/defi-test-app

Regards,

**Comment by Eagle9461** [comment]
https://github.com/tank9995/nexusjs
https://github.com/tank9995/nikolovlazar.com
https://github.com/tank9995/readpilot

How are you?

If you give me a chance, I will show you high responsibility and ability.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by bumpedyou** [comment]
Hello. I am a Abdullah Ahmed from Germany.
As a senior full-stack developer, I have rich of experience in React js and Next js for over 7 years. 
I have ever worked on team and Elevos company before so I am familiar with team work and Github.
I can work on your time zone and with fulltime as you want. 
Email top.dev.g@gmail.com
Looking forward to hearing from you soon.
Thanks

**Comment by bumpedyou** [comment]
Thanks for your reply.
Yes, I am interested in your job, so If you are possible then how about
call with me?
Thanks

**Comment by landmann** [comment]
Hey guys! My name is Nathan and I’m a co-founder at Try it on - [tryitonai.com](http://tryitonai.com/). We’ve gotten some crazy reception recently (magazines, TV shows, book covers, you name it!) and are now looking for our first full-time hire! We need someone that’s a NextJS whiz. I’m the ML engineer and need a front-end sidekick. Pls shoot me an e-mail me if that’s you or you know of someone that’d be interested. nlandmanc@gmail.com

**Comment by stevo2105** [comment]
[Whop](https://whop.com) | Senior Full-Stack Engineer | In person (New York, NY) | Full-time | $170-220k base | [Link](https://jobs.polymer.co/whop/26288)

Whop is a marketplace where people sell access to communities and software they create. We process over $8M / month and have almost a million users. Whop is hiring across the board. We are venture backed and looking for incredibly talented folks to join us in our NYC office. We are located in Brooklyn. 

We have a small engineering team where you will be able to make incredible impact. Please reach out to me on [Twitter](https://twitter.com/cultured) or apply through the link above.

We are only hiring for in person roles at this time so please don't reach out unless you are willing to move to NYC. We have a massive opportunity on our hands and need to be completely aligned!

**Comment by davens** [comment]
[Vochlea Music](www.vochlea.com) | London, UK | Full-time Hybrid | [Jobs](vochlea.com/careers)

Fullstack Engineer £70-£100K
Frontend Engineer £60-£90K

Vochlea Music is a London based creative technology company on a mission to empower music creators everywhere. We’ve created an artificial intelligence (AI) Audio Engine that makes it easy for anyone to create, control and manipulate music using their voice. Imagine beatboxing triggering drum samples, humming a guitar part or controlling effects by the timbre of your voice - all absolutely live!

We are a 16 person startup, now hiring to grow to 20. You would therefore join and have a huge impact in the team. Culturally, we have a relaxed workplace that values your work life balance, and a focus on friendly, fun vibes. We wish to rework our current site and have exciting plans to transform our web presence with community, content and... coolness.  As such we are hiring a new UX designer and also a visual designer for the web team.

Our stack is Typescript, React, Next.js, Node.js, Express and Postgres on AWS. If you're interested please drop us an email with your CV to info@vochlea.co.uk

**Comment by zachselby** [comment]
Hydrow, Inc. &nbsp;|&nbsp; E-commerce Principal Engineer &nbsp;|&nbsp; Boston, MA or Remote &nbsp;|&nbsp; Full-time &nbsp;|&nbsp; [Apply Today](https://jobs.lever.co/Hydrow/aa909810-ad4f-4e6b-ae0f-8bed9a822127)

#### Who We Are
[Hydrow](https://hydrow.com) is the Live Outdoor Reality (LOR)™ rowing machine that delivers on-water experiences at home with workouts led by world-class athletes. If you ask us, we’re the perfect mix of athletes, entrepreneurs, engineers, and creatives. Our mission is not just to create an amazing fitness experience, but a beautiful human experience - one that’s honest, inclusive, and positive. At Hydrow, you’ll have the chance to shape a company that is disrupting the fitness industry.

#### Why We Need You
As an E-commerce Principal Engineer working on the Digital Platforms team, you will lead the engineering behind Hydrow’s direct to consumer web channel on a modern, composable stack. You’ll contribute technical leadership and hands-on execution, establish close partnerships with stakeholders including engineering and marketing leadership and guide ongoing feature development for the web. You will be responsible for driving technical excellence and maintaining modern development practices for internal and partner delivery teams working on our headless digital experience stack (Next.js, Contentful, Shopify).

##### You’re excited about this opportunity because you will…
- Help build an all-new in-house e-commerce development team
- Oversee and drive e-commerce architectural direction for a leading consumer brand
- Work with leading edge e-commerce web technologies (including headless and serverless architecture, edge computing) to design and build beautifully immersive, lightning-fast e-commerce experiences

**Reply by zachselby** [reply]
@Hetape thanks for reaching out. Please use the [Apply Today](https://jobs.lever.co/Hydrow/aa909810-ad4f-4e6b-ae0f-8bed9a822127) link in the post to apply for this position! Look forward to hearing from you.

**Comment by sayjun0505** [comment]
Dear Hiring Manager,

I am writing to express my interest in the position available at your company. As a highly motivated individual with a passion for innovation and problem-solving, I am confident that I have the skills and experience necessary to excel in this role.

My background in developing web project has provided me with the ability to frontend and backend. 
Played an important role on strategic software engineer  , testing and optimization  skilled in full stack development .
I am confident that my work ethic, attention to detail, and ability to work both independently and as part of a team would make me a valuable addition to your team.

Additionally, I am excited about the opportunity to work for a company . This aligns with my personal values and I would be thrilled to contribute to a company that shares my passions.

Thank you for considering my application. I look forward to the opportunity to discuss my qualifications further and how I can contribute to the success of your company.

Sincerely, 
[20230407resume.pdf](https://github.com/vercel/next.js/files/11175147/20230407resume.pdf)

Say Jun

**Comment by Eagle9461** [comment]
Thank you for your application, Excuse me, could you send me your CV?
I’m impressed with your abilities.
I’ll wait for you.

Regards.

**Reply by sayjun0505** [reply]
[20230407resume.pdf](https://github.com/vercel/next.js/files/11176539/20230407resume.pdf)
Here I am.

**Comment by benjaminlee712** [comment]
Thank you for your email.
You can check the attached file to learn more about me.

**Comment by wilbsy** [comment]
UNDEFINEDstate | Senior Full-Stack Engineer | Remote (Americas preferred, worldwide considered) | Contract | Variable based on project | [u12e.com](https://u12e.com/)

We're looking for developers well versed in Next 13, Vercel, Turbo, TS, XState, Storybook, SCSS, and React.  Whether you're experienced in all of the above or highly specialized in only one of the listed skills, there may be a place for regular contract work with us.   

Please reach out using the contact form in the link above (include links to your work, along with the role you filled in any work examples).

**Reply by sayjun0505** [reply]
https://t.me/greengreen12334     live:.cid.7b71ea7b875eed59      
Thank you for your email.
Look forward to working with you.

**Comment by Nightwing627** [comment]
CAYC | Senior Front-End Engineer | Remote (not Indian) | Full-time | Negotiable

Looking for a senior Frontend developer who has rich experience in NextJS, **TypeScript**, **ESlint**, OAuth, Firebase, **Recoil**, SocketIO(client), and Chakra-ui. This project is a bit complex and very urgent so we need only skilled developers who can work full time.

Responsibilities
- Analyzing and Understanding the current project asap
- Strong problem solving skills
- Good communication with collaborators
- Daily updating via git
- Prefer to know about NodeJS, RESTful API, Cyber Security
- Available for working and communicating on PT

We will offer the good salary, and the long term work is available if your performance is very good and complete the project asap.
Let's connect me and discuss more detail.

 !DracoN#7901 on Discord

Sincerely,

CAYC

**Reply by rodinaleksandr0219** [reply]
Hello @Nightwing627 

I am a senior Frontend engineer with more than 7 years of experience.
I am very interested in this position so would like to discuss more.

Regards,
[resume.pdf](https://github.com/vercel/next.js/files/11272603/resume.pdf)

**Reply by codepool867** [reply]
Hi @Nightwing627 
I am so excited to notice you are looking for talented frontend developer.
I have been actively looking for opportunities to boost my experiences I have in web development, and I see high possibility here.
For your information, following is my linkedin profile:
https://linkedin.com/in/jeffrey-childers-tech
Being pretty much interested in your offer, I have just sent you a request on discord.
Hope to hear from you soon.
Jeffrey.

**Reply by mikitam31** [reply]
Hello. Thanks for job posting.

Please check my Linkedin profile kindly
https://www.linkedin.com/in/markin-mikita-8869681bb/

Looking for your reply
Regards,

**Reply by truecode112** [reply]
Hello.
Thanks for your posting.
I am a senior frontend engineer.
You can check on my github.
https://github.com/truecode112
Regards

**Reply by jordan716** [reply]
Hello @Nightwing627

I'm a senior Frontend engineer with 8+ years of rich experience in frontend development using NextJS.
I'm very excited about the opportunity to work in the role since my key skill is NextJS and all the required skills best match my skill set.
I would like to meet you soon and discuss this further.

Email ID: jordanengle716@gmail.com
Phone: 7185142772
[Linkedin](https://www.linkedin.com/in/jordan-engle-front-end-developer)

Thank you,
Best regards,
Jordan Engle

**Reply by spider-yamet** [reply]
How are you?
I love clean code, pixel-perfect and my client's 100% satisfaction best.
To introduce myself, I'm a senior Front end dev with over +7 years EXP and my best skill is JS frameworks such as React/Next, vue.js/Nuxt.js, angular.
Well-versed in numerous programming languages including html5/css3/javascript(ECMA6).
My Skills:
✔️ React (7 years): Redux, Material-UI, Ant Design, Semantic UI, Next.js(SSR, SEO)
✔️ Vue (7 years): StorefrontUI, Vue2, Vue3, Vuex, Nuxt.js(SSR, SEO)
✔️ Shopify (5 years): Online Store(Shopify), Headless Store, Woocommerce, Magento
✔️ Frontend: Html5/Css3, Bootstrap(3,4,5), Tailwind, html template(liquid, blade, twig, ejs)
✔️ Integration: Payment Integration, third party integration(AWS, firebase)

I'm looking for a project to contribute build excellent web with my skills. 
email: needmefordev@gmail.com
Thanks.

**Reply by danielleeas** [reply]
I am very interested in your job post.
I am ready to start your project.
I hope to hear from you soon.
Regards.

**Comment by max00317** [comment]
🙂 Hi, @everyone 
I am a Full Stack developer with 7+ years of experience in web developments.
I have honed my skills over the past 7 years in developing and delivering high-quality software solutions for various clients. My experience has given me the ability to understand the unique requirements of different clients and to work collaboratively with teams to deliver projects on time and within budget. In addition to my experience, I have a deep understanding of various programming languages, frameworks, and tools, which allows me to adapt to new technologies and learn quickly.

☑️Specialized skills:
    -React/Next.js, Vue/Nuxt.js, React Native
    -PHP, Laravel, CodeIgniter
    -HTML/CSS, JavaScript, ECMA6, TypeScript
    -MaterialUI, Tailwind CSS, Bootstrap
    -Node.js, Express.js, API Integration
    -MongoDB, MySQL, PostgreSQL
    -AWS, Firebase, RESTful APIs
    -Git

⚔️ Specific skills:
    -Creativity & Persistence
    -Clear communication
    -Optimizing code & Fast speed
    -keeping deadline

I am ready to start working and available to work 40 hours a week in your time zone.
I'd love to discuss in detail about your project and how I can use my skills to help you attain your goals.

**Comment by webtalent0125** [comment]
Hello there.

I am Austin Lee, Full stack developer who is waiting for a senior software engineer

Location: Singapore
Remote: yes
Willing to relocate: No
Technologies: NextJS, typescript, storybook, Tailwind, Svelte, flux, 100ms, firebase, appwrite, NodeJS, docker, MongoDB, Python
Resume/CV: https://thecrazydev-454b7.web.app/
Email: [talenteddev0125@gmail.com](mailto:talenteddev0125@gmail.com)

**Comment by jonkwheeler** [comment]
https://www.linkedin.com/posts/jonkwheeler_reach-financial-personal-loans-activity-7057365686888615936-cStv?utm_source=share&utm_medium=member_desktop

We've got a bunch of jobs open here at [Reach Financial](https://www.linkedin.com/company/reach-financial/). Come be apart of a huge transformation and help people "outsmart debt for good".

All positions are remote, but some have specific locations we'd prefer you to be based out of.

If you're an Engineer and Toronto based, we want to hear from you. We have openings in this following, most at or above Senior level:

- Cloud/DevOps Architect
- DevOps
- SDET (Software Development Engineer in Test)
- Front End w/ XP in large state based apps
- Software Engineer

We're also looking for Technical Marketing Manager for Product based in the US, and a Project Manager for the Risk / Data team based in the US.

Some of the openings are listed here. Not all. https://lnkd.in/euYKHKQQ

**Comment by JGyroscope** [comment]
Hi everyone nice to meet you.
I kindly read your job post and am very interested in your job post.
So I mastered Vue/Vuex, React/Redux/Saga and Next.js and others.
And then I have so many hands-on experiences with UI/UX design and HTML, and CSS, because I have worked as a Tester and team lead at EvenRise Company for 2 years.
Frontend frameworks such as React and Nextjs with Typescript are all my major priorities and I am ready for anyone.
I also have clean UI building skills with Tailwind CSS and Material UI(Styled components) and rich experience.

And then I have deep knowledge of Git and Nodejs, Mongodb, Mysql, Postgres, Redis, RedShift and PHP.

And my recent work:
(PinPoint) Food delivery and Scheduling website
Next.js + MongoDB + express
https://www.loom.com/share/fb0c84a0cd584a1faa4471848f08a3e7
https://www.loom.com/share/4cf9d4ff90514675a0e76eef3bdc6b9d
https://www.loom.com/share/3bb7fff65d324fc3ac514f017c37b704
https://www.loom.com/share/0be8e084375248cba1969d3b3b1f2df2
https://www.loom.com/share/50f0a5486c9e4d4a959d301a1d22caea
https://www.loom.com/share/6c6d711bcdf44c83b6df4fc6e9722058

https://dynamicbusiness.com/
https://www.loom.com/share/aa42157da544416e825a3a7bf8af0723
Cinnamon(Next.js + Context API + .NET + wordpress plugin)

https://www.cleveroad.com/
Cleveroad(Gatsby.js + GraphQL)

https://www.axattechnologies.com/
Axat(Next.js + Laravel)

https://tilta.com/
Tilta(Vue.js + Django)

https://www.tripadvisor.com/blog/
Tripadvisor(Wordpress)

https://www.igoodi.eu/
IGoodi(Three.js)

https://www.coinfra.io/buy/palm-dao
(Web3 + Solana + Ivory + React + laravel)

Please kindly check my work and if you are interested in me, please kindly DM me and discuss more.

Thanks.

Regards.

**Comment by JGyroscope** [comment]
If you have the telegram, skype or discord account, please tell me.
